hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/ErasureCodingPolicyManager.java

```diff
@@ -482,11 +483,15 @@ private void loadPolicy(ErasureCodingPolicyInfo info) {
    *
    */
   public synchronized void loadPolicies(
-      List<ErasureCodingPolicyInfo> ecPolicies, Configuration conf)
+      List<ErasureCodingPolicyInfo> ecPolicies, Configuration conf,
```

Comment:
Can you leave the old header passing a null so we don't need to modify the tests?

Reply:
+1, keep the original method and parameters as followinng.
`public synchronized void loadPolicies(
      List<ErasureCodingPolicyInfo> ecPolicies, Configuration conf) {
  loadPolicies(ecPolicies, conf, null);
}`